apps/root/src/pages/home/components/portfolio/index.tsx

```diff
@@ -371,21 +373,23 @@ const Portfolio = ({ selectedWalletOption }: PortfolioProps) => {
     setIsRefreshDisabled(true);
     setTimeout(() => setIsRefreshDisabled(false), IntervalSetActions.globalBalance);
     const chains = sdkChains;
-    const addresses = user?.wallets.map(({ address }) => address);
+    const accountId = user?.id;
 
-    if (!addresses) return;
+    if (!accountId) return;
+    const signature = await accountService.getWalletVerifyingSignature({});
 
     await meanApiService.invalidateCacheForBalancesOnWallets({
```

Comment:
lets actually do this throuh the accountService so we dont have to require the user signature here